simple-git-hooks.test.js

```diff
@@ -126,108 +130,122 @@ function getInstalledGitHooks(hooksDir) {
     return result
 }
 
+afterEach(() => {
+    [
+    projectWithConfigurationInAlternativeSeparateJsPath,
+    projectWithConfigurationInAlternativeSeparateCjsPath,
+    projectWithConfigurationInSeparateCjsPath,
+    projectWithConfigurationInSeparateJsPath,
+    projectWithConfigurationInAlternativeSeparateJsonPath,
+    projectWithConfigurationInSeparateJsonPath,
+    projectWithConfigurationInPackageJsonPath,
+    projectWithIncorrectConfigurationInPackageJson,
+    projectWithoutConfiguration,
+    projectWithConfigurationInPackageJsonPath,
+    projectWithUnusedConfigurationInPackageJsonPath,
+    projectWithCustomConfigurationFilePath,
+    ].forEach((testCase) => {
+        removeGitHooksFolder(testCase);
+    });
+});
+
+
 test('creates git hooks if configuration is correct from .simple-git-hooks.js', () => {
     createGitHooksFolder(projectWithConfigurationInAlternativeSeparateJsPath)
 
     spc.setHooksFromConfig(projectWithConfigurationInAlternativeSeparateJsPath)
     const installedHooks = getInstalledGitHooks(path.normalize(path.join(projectWithConfigurationInAlternativeSeparateJsPath, '.git', 'hooks')))
-    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`#!/bin/sh\nexit 1`, 'pre-push':`#!/bin/sh\nexit 1`}))
-
-    removeGitHooksFolder(projectWithConfigurationInAlternativeSeparateJsPath)
+    expect(isEqual(installedHooks, { 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT })).toBe(true);
 })
 
+
 test('creates git hooks if configuration is correct from .simple-git-hooks.cjs', () => {
     createGitHooksFolder(projectWithConfigurationInAlternativeSeparateCjsPath)
 
     spc.setHooksFromConfig(projectWithConfigurationInAlternativeSeparateCjsPath)
     const installedHooks = getInstalledGitHooks(path.normalize(path.join(projectWithConfigurationInAlternativeSeparateCjsPath, '.git', 'hooks')))
-    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`#!/bin/sh\nexit 1`, 'pre-push':`#!/bin/sh\nexit 1`}))
-
-    removeGitHooksFolder(projectWithConfigurationInAlternativeSeparateCjsPath)
+    expect(isEqual(installedHooks, { 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT })).toBe(true);
 })
 
+
 test('creates git hooks if configuration is correct from simple-git-hooks.cjs', () => {
     createGitHooksFolder(projectWithConfigurationInSeparateCjsPath)
 
     spc.setHooksFromConfig(projectWithConfigurationInSeparateCjsPath)
     const installedHooks = getInstalledGitHooks(path.normalize(path.join(projectWithConfigurationInSeparateCjsPath, '.git', 'hooks')))
-    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`#!/bin/sh\nexit 1`, 'pre-push':`#!/bin/sh\nexit 1`}))
-
-    removeGitHooksFolder(projectWithConfigurationInSeparateCjsPath)
+    expect(isEqual(installedHooks, { 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT })).toBe(true);
```

Comment:
I think we can extract `{ 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT }` as variable too